db/migrate/20180219114840_create_restaurants.rb

```diff
@@ -0,0 +1,12 @@
+class CreateRestaurants < ActiveRecord::Migration[5.1]
+  def change
+    create_table :restaurants do |t|
+      t.string :name, null: false, default: ''
```

Comment:
index on name?

Reply:
I will add an index which will also require uniqueness at the db level (https://robots.thoughtbot.com/the-perils-of-uniqueness-validations)